tests/integration/ceph_mgr_test.go

```diff
@@ -0,0 +1,216 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/rook/rook/tests/framework/installer"
+	"github.com/rook/rook/tests/framework/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+// **************************************************
+// *** Mgr operations covered by TestMgrSmokeSuite ***
+//
+// Ceph orchestrator device ls
+// Ceph orchestrator status
+// Ceph orchestrator host ls
+// Ceph orchestrator create OSD
+// Ceph orchestrator ls
+// **************************************************
+func TestMgrSmokeSuite(t *testing.T) {
+	if installer.SkipTestSuite(installer.CephTestSuite) {
+		t.Skip()
+	}
+
+	s := new(MgrSmokeSuite)
+	defer func(s *MgrSmokeSuite) {
+		HandlePanics(recover(), s.op, s.T)
+	}(s)
+	suite.Run(t, s)
+}
+
+type MgrSmokeSuite struct {
+	suite.Suite
+	op        *TestCluster
+	k8sh      *utils.K8sHelper
+	namespace string
+	CLI       MgrCLI
+}
+
+type host struct {
+	Addr     string
+	Hostname string
+	Labels   []string
+	Status   string
+}
+
+type service struct {
+	Container_image_name string
```

Comment:
golang conventions:
- the expected naming is without underscores. For example: `ContainerImageName`
- if the variable is not needed outside the package, it should be private (lower-cased)

Reply:
Using lower case, or not underscores, wasn't possible to parse the values coming from the json. I know it is possible to add json tags ... but maybe i think that probably for the test we can live without them.

Reply:
More common for deserialization would be like this. You get the good golang name and also the json deserialization.
```
ContainerImageName string `json:"Container_image_name"`
```